crmsh/report/core.py

```diff
@@ -441,6 +432,31 @@ def load_context_attributes(context: Context) -> None:
     context.cores_dir_list.extend([constants.COROSYNC_LIB] if os.path.isdir(constants.COROSYNC_LIB) else [])
 
 
+def load_context_trace_dir_list(context: Context) -> None:
+    log_contents = ""
```

Comment:
Better to move these two comment lines (https://github.com/ClusterLabs/crmsh/blob/crmsh-4.6/crmsh/report/collect.py#L192-L193) into here to explain why need to do like this